src/error.ts

```diff
@@ -128,6 +128,9 @@ function isAggregateError(e: unknown): e is Error & { errors: Error[] } {
  * mongodb-client-encryption has a dependency on this error, it uses the constructor with a string argument
  */
 export class MongoError extends Error {
+  get [Symbol.toStringTag]() {
+    return this.name;
+  }
```

Comment:
I think we were using this at one point and stopped -- we should remove it before unnecessarily introducing it